src/main/java/org/humancellatlas/ingest/project/web/SearchFilter.java

```diff
@@ -4,14 +4,22 @@
 import lombok.Builder;
 import lombok.Getter;
 import lombok.ToString;
+import org.humancellatlas.ingest.project.DataAccessTypes;
 
 @AllArgsConstructor
```

Comment:
You have 13 parameters and you are already using the builder pattern. I would suggest to remove the `@AllArgsConstructor` to have a cleaner code.

Reply:
This is needed I believe since SearchFilter is a ModelAttribute and spring needs all of the args available through the constructor

Reply:
@ke4 the `AllArgsConstructor` is needed by Spring when it responds to the web api call and builds the `SearchFilter` parameter to invokes the controller.